doc/classes/Object.xml

```diff
@@ -1121,7 +1121,7 @@
 			Notification received when the object is initialized, before its script is attached. Used internally.
 		</constant>
 		<constant name="NOTIFICATION_PREDELETE" value="1">
-			Notification received when the object is about to be deleted. Can act as the deconstructor of some programming languages.
+			Notification received when the object is about to be deleted. Can act as the destructor of some programming languages.
```

Comment:
If @akien-mga would like to do a pass, this is the final suggestion from @fire 
```suggestion
			Notification received when the object is about to be deleted. Can act as the destructor of some programming languages.
```

Reply:
There's no difference between the two lines in the suggestion?

Reply:
The truth is that I am extremely dumb.
```suggestion
			Notification received when the object is about to be deleted. Can be used like the destructor from some programming languages.
```

Reply:
Seems fine, I'd tweak it a bit further:
```suggestion
			Notification received when the object is about to be deleted. Can be used like destructors in some programming languages.
```
Or:
```suggestion
			Notification received when the object is about to be deleted. Can be used like destructors in object-oriented programming languages.
```
(Provided that's correct terminology, I feel "some" was a bit weird.)